### DISCUSSION TITLE
No icons on neorg with lazy and windows

### DISCUSSION THREAD

**Discussion by darcostan** [nvim-neorg/neorg#1532] [Q&A] [answered]
Hi, I have problem with neorg to enable icons and other conceal things from neorg. 
![image](https://github.com/user-attachments/assets/26f4088d-730b-49e8-be4d-0b233b1aceca)
Some is working like links to other pages: 
![image](https://github.com/user-attachments/assets/8d99b87d-3e9e-4606-84fe-12627ae1524a)
but bullets and TODO is not working.

Config is:
![image](https://github.com/user-attachments/assets/df8d7e08-2181-45ff-bbc3-4fe9d2d766c1)
with lazy neovim.
![image](https://github.com/user-attachments/assets/0eded7cb-aec2-4270-85ca-d8c2dd799e7c)
![image](https://github.com/user-attachments/assets/de73a408-5bc5-4bf5-b0b0-dd9d91535d8b)
Fonts are installed and used in terminal:
![image](https://github.com/user-attachments/assets/dcc75c11-4136-41b7-9c63-3a608bde2615)

Can somebody help me please.
Thanks

**Comment by max397574** [comment, marked as answer]
you need to load the concealer module
look at wiki https://github.com/nvim-neorg/neorg/wiki/Setup-Guide

unrelated to that
you should either use `config=true` or `opts={}`, not `opts=true`

**Reply by darcostan** [reply]
Thank you, this helped a lot. 
This plugin config is working now
![image](https://github.com/user-attachments/assets/927f1c61-3c87-4f51-882c-a7a384f5eee3)
![image](https://github.com/user-attachments/assets/eec5eec2-8ebf-4996-afe6-bdf923555f85)
